pimpmycause/src/sass/styles.scss

```diff
@@ -57,7 +57,7 @@ a {
 }
 
 .content-wrap {
-    max-width: $max-width;
+    max-width: 70em;;
```

Comment:
can we keep it as the variable before? `$max-width;` the variable was updated to be `70em` it is easier to change the variable and not every place we need this change

Reply:
Sorted now.